src/pages/AppList/FiltersAndSorts.ts

```diff
@@ -173,19 +173,8 @@ export const sortAppsItems = (
   unsorted: AppListItem[],
   sortField: SortField<AppListItem>,
   isAscending: boolean
-): Promise<AppListItem[]> => {
-  if (sortField.title === 'Health') {
-    // In the case of health sorting, we may not have all health promises ready yet
-    // So we need to get them all before actually sorting
-    const allHealthPromises: Promise<WithAppHealth<AppListItem>>[] = unsorted.map(item => {
-      return item.healthPromise.then((health): WithAppHealth<AppListItem> => ({ ...item, health }));
-    });
-    return Promise.all(allHealthPromises).then(arr => {
-      return arr.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
-    });
-  }
-  const sorted = unsorted.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
-  return Promise.resolve(sorted);
+): AppListItem[] => {
+  return unsorted.sort(isAscending ? sortField.compare : (a, b) => sortField.compare(b, a));
```

Comment:
...an same comment as the other `FilterAndSort.ts` files. I think the declaration of the `filterBy` function can be fixed to no longer return `Promise<AppListItem[]>`